### PR TITLE
Closing #25 Covidbanner

### LINKED ISSUE #25: Resources Page - Covid Banner
image

### PR DESCRIPTION
**Pull request by jimmyygit**
image
cool

### CODE REVIEW

File: components/ResilienceResource/CovidBanner.module.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+.CovidBanner h1 {
+  text-transform: uppercase;
+  font-weight: bold;
+  margin: 0px 60px;
+  color: white;     
+}
+
+.CovidBanner p {
+  color: white;  
+  text-align: center;
```
Comment (LemonFace0309) [inline components/ResilienceResource/CovidBanner.module.css:15]: might look better to not center the text. I'll ask my sister in the future about it.

File: pages/resilience.js [hunk 2]
```diff
@@ -17,6 +18,9 @@ function resilience() {
           </p>
         </div>
       </TextAndPic>
+
+
+      <CovidBanner />
```
Comment (LemonFace0309) [inline pages/resilience.js:23]: Why so much empty space lol

**Review by LemonFace0309 — APPROVED** [review]

File: components/ResilienceResource/CovidBanner.module.css (hunk 1 above)
Comment (functionstackx) [reply components/ResilienceResource/CovidBanner.module.css:15]: Do you wanna create an issue by referencing this comment so you don't forget? @lemonface0309

File: pages/resilience.js (hunk 2 above)
Comment (functionstackx) [reply pages/resilience.js:23]: Do you wanna create an issue by referencing this comment so you don't forget? @LemonFace0309

File: components/ResilienceResource/CovidBanner.module.css (hunk 1 above)
Comment (functionstackx) [reply components/ResilienceResource/CovidBanner.module.css:15]: Click the 3 dots and there is an option called "reference in new issue"

Comment (jimmyygit) [reply components/ResilienceResource/CovidBanner.module.css:15]: ah ok. I just followed the Figma

File: pages/resilience.js (hunk 2 above)
Comment (jimmyygit) [reply pages/resilience.js:23]: Lol idk, I guess I was anticipating other components to go there

**Pull request merged**